action.yaml

```diff
@@ -438,16 +438,22 @@ runs:
         cat .frogbot/frogbot-config.yml
```

Comment:
Do we still need this configuration file?

Reply:
No We don't need this file

Reply:
So this step is going to be removed?